taca_ngi_pipeline/deliver/deliver.py

```diff
@@ -87,6 +87,12 @@ def __init__(self, projectid, sampleid, **kwargs):
         self.reportpath = getattr(self, 'reportpath', None)
         self.force = getattr(self, 'force', False)
         self.stage_only = getattr(self, 'stage_only', False)
+        #Fetches a project name, should always be availble; but is not a requirement
+        try:
+            n = db.project_entry(db.dbcon(), projectid)['name']
+            self.projectname = n
+        except KeyError:
+            pass
```

Comment:
if you don't mind the object having the attribute `projectname` but that it can be None, I'd suggest to simplify the above code to `self.projectname = getattr(db.project_entry(db.dbcon(), projectid), 'name', None)`.. or better yet, just set it to None and do the db lookup when the value is needed